tests/mypy_test.py

```diff
@@ -131,6 +131,7 @@ def main():
             runs += 1
             flags = ['--python-version', '%d.%d' % (major, minor)]
             flags.append('--strict-optional')
+            flags.append('--no-site-packages')
```

Comment:
Perhaps this should be gated by Python version?

Reply:
Do you mean that we should do this only for Python versions for which we don't have a binary? I don't think that's worth it, since for typeshed's purposes we don't want to look at site-packages in any case. Actually, our CI might get confused when PEP 561-enabled packages start showing up in the venv we use to run tests.

Reply:
Agreed.

Reply:
Fair point. I think this change LGTM then.